src/Core/Commander/Interfaces/Control/Control.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * Generated On: 2016-11-22
+ * Class: Controls
+ * Description: Cette Classe construit un control.
+ */
+
+import ApiGlobe from 'Core/Commander/Interfaces/ApiInterface/ApiGlobe';
```

Comment:
Not used anywhere. Also, cyclic dependencies should be avoided as much as possible (it creates difficult-to-maintain code).